module/programs/helper.js

```diff
@@ -298,20 +298,18 @@ module.exports = class ProgramsHelper {
 
         // If the request body has roles set to "all", add "all" as a role; otherwise, get the roles from userRoles to update scope.roles
         if (Array.isArray(roles) && roles.length > 0) {
-          let currentRoles = await programsQueries.programDocuments({ _id: programId }, ['scope.roles'])
-					currentRoles = currentRoles[0].scope.roles
+          let currentRoles = await programsQueries.programDocuments({ _id: programId }, ['scope.roles']);
```

Comment:
can you do proper variable naming ? instead of fetching let currentRoles = on line 301 
let programDocumentRecord =  await programsQueries.programDocuments({ _id: programId }, ['scope.roles']);

Reply:
Added @borkarsaish65